packages/vuetify/src/mixins/overlayable/index.ts

```diff
@@ -132,7 +120,9 @@ export default Vue.extend<Vue & Toggleable & Stackable & options>().extend({
           ['INPUT', 'TEXTAREA', 'SELECT'].includes((e.target as Element).tagName) ||
           // https://github.com/vuetifyjs/vuetify/issues/4715
           (e.target as HTMLElement).isContentEditable
-        ) return
+        ) {
+          return
+        }
```

Comment:
Please revert unrelated styling changes.

Reply:
Oops, sorry - didn't catch my linter doing that. I'll revert any styling changes it made!